app/models/ncj-job-template.ts

```diff
@@ -26,8 +26,8 @@ export interface NcjJobTemplate {
 
 export interface NcjPoolTemplate {
     parameters: StringMap<NcjParameter>;
-    pool: NcjJobConfiguration;
-    variables: NcjJobConfiguration;
+    variables?: StringMap<any>;
+    pool: StringMap<string>;
 }
```

Comment:
@timotheeguerin, can you double check this change. it seemed wrong and like a copy/paste issue. didn't matter with binding raw JSON to it, but the tests picked it up.